library/core/src/fmt/mod.rs

```diff
@@ -333,7 +333,7 @@ pub struct Arguments<'a> {
 #[unstable(feature = "fmt_internals", issue = "none")]
 impl<'a> Arguments<'a> {
     #[inline]
-    #[rustc_const_unstable(feature = "const_fmt_arguments_new", issue = "none")]
+    #[rustc_const_stable(feature = "const_fmt_arguments_new", since = "CURRENT_RUSTC_VERSION")]
```

Comment:
Not sure if this warrants a separate stabilization PR because it's an internal only API behind `[feature(fmt_internals)]`.